.travis.yml

```diff
@@ -106,7 +108,7 @@ jobs:
     language: python
     python:
       - "3.8"
-    if: (branch = master OR type = pull_request) AND fork = false
+    if: branch = master OR tag =~ /^release.*$/
```

Comment:
we would like to run that tests also on feature branches (function tests also)

Reply:
Done.